app/views/words/index.html.erb

```diff
@@ -0,0 +1,48 @@
+<%= provide(:title, "Words") %>
+<div class="row mt-5 mx-auto justify-content-center">
+    <%= render 'menu' %>
+        <div class="ml-5">
+            <div class="card text-primary bg-light" style="width: 60rem;">
+                <div class="row card-header">
+                    <h3 class="offset-1">Words for the lessons</h3>
```

Comment:
to make identifying the selected category, you can replace this code to this:
`<h3 class="offset-1">Words for the <%= @category.title %> lessons</h3>`
and add this to your words controller:
`@category = Category.find(params[:category_id])`